package-lock.json

```diff
@@ -9720,42 +9730,105 @@
       }
     },
     "@testing-library/dom": {
-      "version": "7.16.2",
-      "resolved": "https://registry.npmjs.org/@testing-library/dom/-/dom-7.16.2.tgz",
-      "integrity": "sha512-4fT5l5L+5gfNhUZVCg0wnSszbRJ7A1ZHEz32v7OzH3mcY5lUsK++brI3IB2L9F5zO4kSDc2TRGEVa8v2hgl9vA==",
+      "version": "7.28.1",
```

Comment:
Since we didn't have a change to the corresponding `package.json` file we can probably revert these `package-lock.json` changes as I'm guessing they are a remnant of the package tomfoolery we were going through yesterday.